tag/nitag.yml

```diff
@@ -0,0 +1,1816 @@
+        - in: path
+          name: id
+          description: Selection ID
+          type: string
+          required: true
+      responses:
+        200:
+          description: Success
+          schema:
+            type: array
+            items:
+              description: Tag path with timestamped value
+              type: object
+              title: Tag Path With Current Value
+              properties:
+                path:
+                  type: string
+                  description: Tag path
+                current:
+                  $ref: '#/definitions/TimestampedTagValue'
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          description: Selection with the given ID was not found
+        default:
+          $ref: '#/responses/Error'
+  /v2/selections/{id}/values/current/value:
+    get:
+      tags:
+        - selections
+      summary: Get the current values of the tags in a selection, excluding timestamps
+      description: Returns the current values of the tags in a selection, excluding timestamps
+      operationId: GetSelectionCurrentValues
+      parameters:
+        - in: path
+          name: id
+          description: Selection ID
+          type: string
+          required: true
+      responses:
+        200:
+          description: Success
+          schema:
+            type: array
+            items:
+              description: Tag path with value
+              type: object
+              title: Tag Path With Value
+              properties:
+                path:
+                  type: string
+                  description: Tag path
+                current:
+                  $ref: '#/definitions/TagValue'
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          description: Selection with the given ID was not found
+        default:
+          $ref: '#/responses/Error'
+  /v2/selections/{id}/values/current/timestamp:
+    get:
+      tags:
+        - selections
+      summary: Get the timestamps of the current values of the tags in a selection
+      description: Gets the timestamps of the current values of the tags in a selection
+      operationId: GetSelectionValuesCurrentTimestamp
+      parameters:
+        - in: path
+          name: id
+          description: Selection ID
+          type: string
+          required: true
+      responses:
+        200:
+          description: Success
+          schema:
+            description: Array containing the paths of the tags in the selection, along with the timestamp of their current values
+            type: array
+            title: Tag Paths With Timestamps
+            items:
+              type: object
+              title: Tag Path and Timestamp
+              description: A tag path and the timestamp of its current value
+              properties:
+                timestamp:
+                  description: Timestamp associated with the current value
+                  type: string
```

Comment:
For timestamps, it could be useful to give an example so it's easy to tell the expected format. For test monitor, we've documented our timestamps via `format: iso-date-time` which isn't a standard format, but the format property is extensible.

Reply:
Added an example